game/main.cc

```diff
@@ -291,24 +248,16 @@ template <typename Container> void confOverride(Container const& c, std::string
 
 void outputOptionalFeatureStatus();
 
-void fatalError(std::string msg, bool hasLog = false, std::string title = "FATAL ERROR") {
-	std::ostringstream errMsg;
-	errMsg << msg;
-	if (hasLog) {
-		errMsg << std::endl << "More details might be available in " << getLogFilename() << ".";
-	}
-	errMsg << std::endl << "If you think this is a bug in Performous, please report it at "
-	  << std::endl << "  https://github.com/performous/performous/issues";
-	SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, title.c_str(),
-	  errMsg.str().c_str(), nullptr);
+static void fatalError(const std::string &msg) {
+	auto errMsg = msg + "\nIf you think this is a bug in Performous, please report it at \n"
+	                    "  https://github.com/performous/performous/issues";
+        auto title = "FATAL ERROR";
+	SDL_ShowSimpleMessageBox(SDL_MESSAGEBOX_ERROR, title, errMsg.c_str(), nullptr);
 	std::cerr << title << ": " << msg << std::endl;
-	if (hasLog) {
-		std::clog << "core/error: " << errMsg.str() << std::endl;
-	}
+        std::clog << "core/error: " << errMsg << std::endl;
```

Comment:
I think this line is also indented too far

Reply:
done